### PR TITLE
Adding digvis.co.uk

### PR DESCRIPTION
**Pull request by ViktorasJucikas**

### CODE REVIEW

File: spammers.txt [hunk 1]
```diff
@@ -38,6 +38,7 @@
 @devpeople.co.uk OR 
 @digitalgurus.co.uk OR 
 @digitalpursuit.co.uk OR 
+@digvis.co.uk OR
```
Comment (nelsonic) [inline spammers.txt]: @ViktorasJucikas space required at end of line else it breaks the filter.

**Comment by nelsonic** [comment]
@ViktorasJucikas did these guys send you irrelevant ("*spam*") email?

**Comment by ViktorasJucikas** [comment]
Yep, both me and my team.

**Comment by nelsonic** [comment]
Oh... that's a shame. :disappointed: 
Did you try to ask them to remove you from their list...?

**Comment by ViktorasJucikas** [comment]
Many times... ;(

**Comment by nelsonic** [comment]
Well, you *tried* to help them *not* spam you. 
Sad they did not listen.

Nice Blog by the way ... http://2founders.com :+1:

**Pull request merged**